code/datums/components/rot.dm

```diff
@@ -1,74 +0,0 @@
-/datum/component/rot
-	/// Amount of miasma we're spawning per tick
-	var/amount = 1
-	/// Time remaining before we remove the component
-	var/time_remaining = 5 MINUTES
-
-/datum/component/rot/Initialize(new_amount)
-	if(!isatom(parent))
-		return COMPONENT_INCOMPATIBLE
-
-	if(new_amount)
-		amount = new_amount
-
-	START_PROCESSING(SSprocessing, src)
-
-/datum/component/rot/Destroy(force, silent)
-	STOP_PROCESSING(SSprocessing, src)
-	. = ..()
-
-/datum/component/rot/process(delta_time)
-	var/atom/A = parent
-	
-	//SSprocessing goes off per 1 second
-	time_remaining -= delta_time * 1 SECONDS
-	if(time_remaining <= 0)
-		qdel(src)
-		return
-
-	var/turf/open/T = get_turf(A)
-	if(!istype(T) || T.planetary_atmos || T.return_air().return_pressure() > (WARNING_HIGH_PRESSURE - 10))
```

Comment:
I need to ask someone if this actually helps the processing of the game because due to the defines the component was apparently not producing any miasma but the process that is being called every decisecond is all of these procs.